src/components/SlateEditor/plugins/contactBlock/ContactBlockForm.tsx

```diff
@@ -0,0 +1,205 @@
+  blobColor?: 'pink' | 'green';
+  blob?: 'pointy' | 'round';
+  metaImageId?: string;
+}
+
+const rules: RulesType<ContactBlockFormValues> = {
+  jobTitle: {
+    required: true,
+  },
+  description: {
+    required: true,
+  },
+  name: {
+    required: true,
+  },
+  email: {
+    required: true,
+  },
+  metaImageId: {
+    required: true,
+  },
+};
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  gap: ${spacing.small};
+`;
+
+interface Props {
+  initialData?: ContactBlockEmbedData;
+  onSave: (data: ContactBlockEmbedData) => void;
+  onCancel: () => void;
+}
+
+const inputStyle = css`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+`;
+
+const inputContentStyle = css`
+  ${inputStyle};
+  height: 130px;
+`;
+
+const StyledFormikField = styled(FormikField)`
+  margin: 0px;
+  padding-bottom: ${spacing.small};
+`;
+
+const toInitialValues = (initialData?: ContactBlockEmbedData): ContactBlockFormValues => {
+  return {
+    resource: 'contact-block',
+    jobTitle: initialData?.jobTitle ?? '',
+    blobColor: initialData?.blobColor ?? 'green',
+    description: initialData?.description ?? '',
+    blob: initialData?.blob ?? 'pointy',
+    metaImageId: initialData?.imageId,
+    name: initialData?.name ?? '',
+    email: initialData?.email ?? '',
+  };
+};
+const blobTypes = ['pointy', 'round'];
+const blobColors = ['green', 'pink'];
+
+const ContactBlockForm = ({ initialData, onSave, onCancel }: Props) => {
+  const { t } = useTranslation();
+  const initialValues = useMemo(() => toInitialValues(initialData), [initialData]);
+  const initialErrors = useMemo(() => validateFormik(initialValues, rules, t), [initialValues, t]);
+
+  const onSubmit = useCallback(
+    (values: ContactBlockFormValues) => {
+      if (!values.metaImageId) {
+        return;
+      }
+      const newData: ContactBlockEmbedData = {
+        resource: 'contact-block',
+        imageId: values.metaImageId,
+        jobTitle: values.jobTitle,
+        description: values.description,
+        name: values.name,
+        blob: values.blob,
+        blobColor: values.blobColor,
+        email: values.email,
+      };
+      onSave(newData);
+    },
+    [onSave],
+  );
+
+  return (
+    <Formik
+      initialValues={initialValues}
+      initialErrors={initialErrors}
+      onSubmit={onSubmit}
+      validateOnMount
+      validate={(values) => validateFormik(values, rules, t)}
+    >
+      {({ dirty, isValid, handleSubmit }) => (
+        <>
+          <StyledFormikField name="name" showError>
+            {({ field }: FieldProps) => <InputV2 label={t('form.name.name')} {...field} />}
+          </StyledFormikField>
+          <StyledFormikField name="jobTitle" showError>
+            {({ field }: FieldProps) => (
+              <InputV2 css={inputStyle} label={t('form.name.jobTitle')} {...field} />
+            )}
+          </StyledFormikField>
+          <StyledFormikField name="email" showError>
```

Comment:
Kan like gjerne bruke typer om det gir verdi.
```suggestion
          <StyledFormikField name="email" type="email" showError>
```